### ISSUE TITLE
Graylog v1.1.0-beta.2 - Unknown command omnibus-ctl: command (subcommand)

### ISSUE THREAD

**Issue by mariussturm** [Graylog2/graylog2-images#64]
Since I upgraded the virtual appliance (1.0) to Graylog v1.1.0-beta.2 with the omnibus package, when I issue the "graylog-ctl" command it says there is an unknown command:

Here is the message:

```
ubuntu@graylog:/tmp$ graylog-ctl
I don't know that command.
/opt/graylog/embedded/bin/omnibus-ctl: command (subcommand)
```

**Comment by mariussturm** [comment]
Sorry, can not reproduce with an upgrade from 1.0.2 -> 1.1.0-beta.3. Could please try the latest beta?
If the error still persist, could post the following:
`ls -l /usr/bin/graylog-ctl`
`ls -l /opt/graylog/embedded/service/omnibus-ctl`

**Comment by mariussturm** [comment]
@reivier77 ping

**Comment by reivier77** [comment]
I upgraded the appliance with the omnibus package like this: 1.0->1.1.0-beta2->1.1.0-beta3
I still have the issue.

Here is the request ouput:

ubuntu@graylog:/tmp$ ls -l /usr/bin/graylog-ctl
lrwxrwxrwx 1 root root 37 May 28 07:16 /usr/bin/graylog-ctl -> /opt/graylog/embedded/bin/graylog-ctl
ubuntu@graylog:/tmp$ ls -l /opt/graylog/embedded/service/omnibus-ctl
total 60
-rw-r--r-- 1 root root  523 May 27 10:31 enforce_ssl.rb
-rw-r--r-- 1 root root  900 May 27 10:31 list_servers.rb
-rw-r--r-- 1 root root  528 May 27 10:31 local_connect.rb
-rw-r--r-- 1 root root 1279 May 27 10:31 reconfigure_as_backend.rb
-rw-r--r-- 1 root root 1269 May 27 10:31 reconfigure_as_datanode.rb
-rw-r--r-- 1 root root 1266 May 27 10:31 reconfigure_as_server.rb
-rw-r--r-- 1 root root 1287 May 27 10:31 reconfigure_as_webinterface.rb
-rw-r--r-- 1 root root  717 May 27 10:31 set_admin_password.rb
-rw-r--r-- 1 root root  695 May 27 10:31 set_admin_username.rb
-rw-r--r-- 1 root root 1145 May 27 10:31 set_cluster_master.rb
-rw-r--r-- 1 root root 1871 May 27 10:31 set_email_config.rb
-rw-r--r-- 1 root root  602 May 27 10:31 set_node_id.rb
-rw-r--r-- 1 root root 1645 May 27 10:31 set_retention.rb
-rw-r--r-- 1 root root  678 May 27 10:31 set_server_secret.rb
-rw-r--r-- 1 root root  835 May 27 10:31 set_timezone.rb

**Comment by mariussturm** [comment]
This looks all fine. When you actually apply a subcommand like `sudo graylog-ctl status` what do you get then?

**Comment by reivier77** [comment]
ubuntu@graylog:/tmp$ sudo graylog-ctl status
run: elasticsearch: (pid 23624) 10223s; run: log: (pid 1607) 171668s
run: etcd: (pid 23648) 10222s; run: log: (pid 1630) 171668s
run: graylog-server: (pid 24364) 10177s; run: log: (pid 1605) 171668s
run: graylog-web: (pid 23780) 10217s; run: log: (pid 1629) 171668s
run: mongodb: (pid 23837) 10216s; run: log: (pid 1606) 171668s
run: nginx: (pid 23844) 10216s; run: log: (pid 1631) 171668s

Actually all the subcommands work. I only noticed this anomaly when I typed "sudo graylog-ctl" which wasn't here before the upgrade:

ubuntu@graylog:/tmp$ sudo graylog-ctl
I don't know that command.
/opt/graylog/embedded/bin/omnibus-ctl: command (subcommand)
cleanse
  Delete *all* graylog data, and start from scratch.
enforce-ssl
  Limit web interface to SSL access
graceful-kill
  Attempt a graceful stop, then SIGKILL the entire process group.
help
  Print this help message.
hup
  Send the services a HUP.
int
  Send the services an INT.
kill
  Send the services a KILL.
list-servers
  List all Graylog servers in your cluster
local-connect
  Connect services through localhost
once
  Start the services if they are down. Do not restart them if they stop.
reconfigure
  Reconfigure the application.
reconfigure-as-backend
  Run Graylog server and Elasticsearch on this node
reconfigure-as-datanode
  Run Elasticsearch on this node only
reconfigure-as-server
  Run Graylog server on this node only
reconfigure-as-webinterface
  Run Graylog web interface and MongoDB on this node
restart
  Stop the services if they are running, then start them again.
service-list
  List all the services (enabled services appear with a *.)
set-admin-password
  Override admin password
set-admin-username
  Set username for administrator
set-cluster-master
  Set IP address of your cluster master server
set-email-config
  Setup email configuration
set-node-id
  Set server node ID
set-retention
  Setup storage configuration
set-server-secret
  Override server secret token
set-timezone
  Set timezone your server is in
show-config
  Show the configuration that would be generated by reconfigure.
start
  Start services if they are down, and restart them if they stop.
status
  Show the status of all the services.
stop
  Stop the services, and do not restart them.
tail
  Watch the service logs of all enabled services.
term
  Send the services a TERM.
uninstall
  Kill all processes and uninstall the process supervisor (data will be preserved).
